tcf_website/templates/course/course_professor.html

```diff
@@ -4,72 +4,198 @@
+                    <canvas width="100%" height="100%" id="myChart"></canvas>
+                </div>
+                <div class="absolute-center">
+                    <p class="gpa-text"> No grades found <i class="far fa-frown-open fa-fw"></i></p>
+                    <p class="students-text"> &mdash; Students</p>
+                </div>
             </div>
+
         </div>
-        {% endif %}
-        {% if hours %}
-        <div class="card col-lg-3 text-center">
-            <div class="card-body">
-              <h5 class="card-title display-4">
-                  {{ hours|floatformat:2 }}
-              </h5>
-              <p class="card-text"><i class="fa fa-hourglass-half fa-fw"></i>&nbsp;Hours/Week</p>
+
+
+        <div class="dropdown">
+            <button id="review-sort-select" class="btn btn-secondary dropdown-toggle" type="button" id="sortOptions"
+                    data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></button>
+            <div class="dropdown-menu" aria-labelledby="sortOptions">
+                <button id="votes-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">Most
+                    Helpful
+                </button>
+                <button id="recent-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">Most
+                    Recent
+                </button>
+                <button id="highrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
+                    Highest
+                    Rating
+                </button>
+                <button id="lowrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
+                    Lowest
+                    Rating
+                </button>
             </div>
         </div>
-        {% endif %}
-    </div>
-    {% if reviews %}
-    <div class="dropdown">
-      <button id="review-sort-select" class="btn btn-secondary dropdown-toggle" type="button" id="sortOptions" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></button>
-      <div class="dropdown-menu" aria-labelledby="sortOptions">
-        <button id="votes-sort-btn" class="dropdown-item" type="button">Most Helpful</button>
-        <button id="recent-sort-btn" class="dropdown-item" type="button">Most Recent</button>
-        <button id="highrating-sort-btn"class="dropdown-item" type="button">Highest Rating</button>
-        <button id="lowrating-sort-btn"class="dropdown-item" type="button">Lowest Rating</button>
-      </div>
-    </div>
-    {% endif %}
 
-    {% include "reviews/reviews.html" with reviews=reviews %}
+        {% include "reviews/reviews.html" with reviews=reviews %}
     </div>
 {% endblock %}
 
 {% block js %}
-<script type="module" src="{% static 'reviews/review.js' %}"></script>
-{% endblock %}
+    <script type="module" src="{% static 'reviews/review.js' %}"></script>
```

Comment:
is this needed if chart.js is in npm package.json

Reply:
no idea